File: 412-fizz-buzz/fizz-buzz.java
class Solution {
    public List<String> fizzBuzz(int n) {
        String str;
        List<String> result = new ArrayList<>();
        for (int i = 1; i <= n; i++) {
            str = "";
            if (i % 3 == 0)
                str += "Fizz";
            if (i % 5 == 0)
                str += "Buzz";
            if (str == "")
                str = String.valueOf(i);
            result.add(str);
        }
        return result;
    }
}

File: 412-fizz-buzz/fizz-buzz.cpp
class Solution {
public:
    vector<string> fizzBuzz(int n) {
        vector<string> result;
        string fizz = "Fizz", buzz = "Buzz", add;
        for (int i = 1; i <= n; i++) {
            add = "";
            if (i % 3 == 0) {
                add += fizz;
            }
            if (i % 5 == 0) {
                add += buzz;
            }
            if (add == "") {
                add = to_string(i);
            }
            result.push_back(add);
        }
        return result;
    }
};